### ISSUE TITLE
No response when double clicking on AION Wallet v.2 Windows

### ISSUE THREAD

**Issue by bisakjee** [aionnetwork/Desktop-Wallet#233]

**Comment by mohnjatthews** [comment]
What operating system are you running @bisakjee?

**Comment by bisakjee** [comment]
MS Windows 10 home 64 bit



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by mohnjatthews** [comment]
Would you be able to post your log file for the Aion Desktop Wallet? Instructions for [how to find your logs in Windows 10 can be found here](https://docs.aion.network/docs/troubleshooting#section-windows).

**Comment by bisakjee** [comment]
Hi

Sure, the log file contains the following:

java.lang.UnsupportedClassVersionError: org/aion/wallet/WalletApplication has been compiled by a more recent version of the Java Runtime (class file version 54.0), this version of the Java Runtime only recognizes class file versions up to 52.0
           at java.lang.ClassLoader.defineClass1(Native Method)
           at java.lang.ClassLoader.defineClass(Unknown Source)
           at java.security.SecureClassLoader.defineClass(Unknown Source)
           at java.net.URLClassLoader.defineClass(Unknown Source)
           at java.net.URLClassLoader.access$100(Unknown Source)
           at java.net.URLClassLoader$1.run(Unknown Source)
           at java.net.URLClassLoader$1.run(Unknown Source)
           at java.security.AccessController.doPrivileged(Native Method)
           at java.net.URLClassLoader.findClass(Unknown Source)
           at java.lang.ClassLoader.loadClass(Unknown Source)
           at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
           at java.lang.ClassLoader.loadClass(Unknown Source)
           at sun.launcher.LauncherHelper.checkAndLoadMain(Unknown Source)
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main"

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by centrys-otniel** [comment]
This may sound stupid, but have you ever started the wallet by opening the C:\Program Files\AionWallet folder and starting the AionWallet.exe from there?

**Comment by bisakjee** [comment]
Hi

I dont think its even installing properly.

In the C drive under the users/username/.aion folder, there only appears one folder which is the log file with the following error



java.lang.UnsupportedClassVersionError: org/aion/wallet/WalletApplication has been compiled by a more recent version of the Java Runtime (class file version 54.0), this version of the Java Runtime only recognizes class file versions up to 52.0
           at java.lang.ClassLoader.defineClass1(Native Method)
           at java.lang.ClassLoader.defineClass(Unknown Source)
           at java.security.SecureClassLoader.defineClass(Unknown Source)
           at java.net.URLClassLoader.defineClass(Unknown Source)
           at java.net.URLClassLoader.access$100(Unknown Source)
           at java.net.URLClassLoader$1.run(Unknown Source)
           at java.net.URLClassLoader$1.run(Unknown Source)
           at java.security.AccessController.doPrivileged(Native Method)
           at java.net.URLClassLoader.findClass(Unknown Source)
           at java.lang.ClassLoader.loadClass(Unknown Source)
           at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
           at java.lang.ClassLoader.loadClass(Unknown Source)
           at sun.launcher.LauncherHelper.checkAndLoadMain(Unknown Source)
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main"

Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by centrys-otniel** [comment]
That is local data storage. It's populated only after startup of the application.
The install is in C:\Program Files\AionWallet. Is there anything there?

**Comment by bisakjee** [comment]
Yes, the AionWallet folder is populated, but double clicking the exe file doesnt do anything.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by centrys-otniel** [comment]
Ok. It was a long shot.
Basically the error says that its using an older java version than the one it should.

Can you try to uninstall any Java runtime environment or java development kit that might be installed on your computer and try again? 

The correct one should be in the wallet's install folder. Don't remove that :)

**Comment by bisakjee** [comment]
Ok thanks.

Do i need to re-install java from somewhere once i uninstall it?



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by uipe** [comment]
Hey, I am having a very similar issue, click in the icon and nothing happens.
Tried in Windows 10 but do not have even the log files being created. 

I also tried in a Windows 7 machine and in that case the error I get in the log files is:
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/C:/Program%20Files/AionWallet/lib/libAionApi-v0.1.14.b5ef3b8-18-10-11.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/C:/Program%20Files/AionWallet/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ILLEGAL_INSTRUCTION (0xc000001d) at pc=0x0000000062541a70, pid=2328, tid=3132
#
# JRE version: OpenJDK Runtime Environment (11.0.1+13) (build 11.0.1+13)
# Java VM: OpenJDK 64-Bit Server VM (11.0.1+13, mixed mode, tiered, compressed oops, serial gc, windows-amd64)
# Problematic frame:
# C  [libblake2b.dll+0x1a70]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\UTILIZ~1\AppData\Local\Temp\\hs_err_pid2328.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

**Comment by centrys-otniel** [comment]
No need there is a java already packed with the wallet.

It should be on c:/Program Files/AionWallet/java/bin/java.exe

**Comment by centrys-otniel** [comment]
Hi @uipe and thanks for pitching in.

1. The SLF4J is only a warning, nothing fatal
2. Win7 is not officially supported, however that's not necesarily a problem (I myself am using the wallet on a Win7 machine). It could be that the Win7 machine is just an older processor and it's missing an instruction set used by libblake2b which is a native libary used for hashing. Could you maybe open a separate issue in which you include this error and preferably the crash log file referenced in this error (C:\Users\UTILIZ~1\AppData\Local\Temp\hs_err_pid2328.log)

**Comment by bisakjee** [comment]
Sorry no luck so far.

1. Uninstalled java.
2. Rebooted.
3. Re-installed the 1.2.2 package.
4. Same issue

Nothing on the log file.


Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by bisakjee** [comment]
See screenshot i was getting when checking java version before i uninstalled java



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by centrys-otniel** [comment]
I don't see a screenshot

**Comment by bisakjee** [comment]
Yes sure whats the username i am looking for?



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by centrys-otniel** [comment]
Otniel Nicola

**Comment by bisakjee** [comment]
Hi

If you install the latest wallet, this will resolve your issue.



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by paco-marvin** [comment]
I have the latest version...downloaded (again) this morning along with Java versions from this website.

**Comment by paco-marvin** [comment]
Sorry, I originally stated that I had v1.2.0 but I actually have the most recent v1.2.2

**Comment by paco-marvin** [comment]
Problem still exists. Here is a copy of the latest Aion log file. Hope this helps.
    SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/C:/Program%20Files/AionWallet/lib/libAionApi-v0.1.14.b5ef3b8-18-10-11.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/C:/Program%20Files/AionWallet/lib/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [ch.qos.logback.classic.util.ContextSelectorStaticBinder]
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ILLEGAL_INSTRUCTION (0xc000001d) at pc=0x0000000062541a70, pid=308, tid=9792
#
# JRE version: OpenJDK Runtime Environment (11.0.1+13) (build 11.0.1+13)
# Java VM: OpenJDK 64-Bit Server VM (11.0.1+13, mixed mode, tiered, compressed oops, g1 gc, windows-amd64)
# Problematic frame:
# C  [libblake2b.dll+0x1a70]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\Marvin\AppData\Local\Temp\\hs_err_pid308.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

Again, the Java version I used was downloaded from the Installation Instructions accompanying the Aion 1.2.2 download on your web site (both 8 and 16 bit versions were downloaded, installed, and the computer was restarted).
Now what?

**Comment by paco-marvin** [comment]
Sorry again..... that's both 32 and 64 bit versions of Java

**Comment by centrys-otniel** [comment]
@qoire let's close this issue. It seems like there's a new issue arising with the native blake2b library.

@paco-marvin , can you please open a new ticket and include the crash-log from `C:\Users\Marvin\AppData\Local\Temp\hs_err_pid308.log` ?
